### DISCUSSION TITLE
Block Style Version Changer

### DISCUSSION THREAD

**Discussion by thisiseiuai** [ScratchAddons/ScratchAddons#3799] [Ideas]
basically it changes Scratch blocks to the style of previous versions of Scratch, like 2.0 with the shading, and 1.4 with the pixelated style

this could also be used with the color and shape changer but idk how hard that would be (I'd imagine that color would be easier than shape)

**Comment by WorldLanguages** [comment]
Nice username :P

**Reply by thisiseiuai** [reply]
I made this account to suggest ideas I have

**Comment by Mbrick2** [comment]
Custom Block Shapes + Colors

**Reply by thisiseiuai** [reply]
but what about the borders? the 2.0 blocks had shading, or am I reading this wrong?

**Comment by mxmou** [comment]
I don't think blocks were pixelated intentionally in 1.4, so changing the border colors and shape of notches would be enough for that version.

**Reply by thisiseiuai** [reply]
the main reason I included 1.4 was because of the pixelated blocks, but maybe that could be a separate setting

**Comment by thisiseiuai** [comment]
bump? idk this is the first thing I did when I made this account
